Gemfile

```diff
@@ -45,7 +50,8 @@ gem 'delayed_job_active_record'
 gem 'daemons'
 gem "validate_url"
 gem "twitter"
-gem "atdis"
+# Can revert to plain gem when this PR is merged https://github.com/openaustralia/atdis/pull/39
+gem "atdis", github: "openaustralia/atdis", branch: "activemodel4"
```

Comment:
@henare I’ve released a new version of the atdis gem with your PR merged in. Could you fix the hack here?